convert2rhel/pkgmanager/__init__.py

```diff
@@ -15,20 +15,25 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+
```

Comment:
Another future note for me: This might get merge conflicts if https://github.com/oamg/convert2rhel/pull/713 is merged first.